test/functional/ui/screen_concealing_spec.lua

```diff
@@ -0,0 +1,144 @@
+local helpers = require('test.functional.helpers')
+local Screen = require('test.functional.ui.screen')
+local clear, feed, execute = helpers.clear, helpers.feed, helpers.execute
+local insert = helpers.insert
+
+describe('Screen', function()
+  local screen   
+  
+  describe("Concealing multiple '&&' With 'syn-match'", function()
+    before_each(function()
+      clear()
+      screen = Screen.new()
+      screen:attach()
+      screen:set_default_attr_ignore( {{}, {bold=true, foreground=255}} ) 
+      insert([[
+        &&
+        &&
+        &&
+        &&
+        &&
+        &&
+        ]])
+      screen:expect([[
```

Comment:
Can you explain why `expect()` is needed in `before_each()`? Hopefully we can trust that `clear()` and `Screen.new()` is doing its job :)